dice-game-ui-angular/src/styles.scss

```diff
@@ -1,4 +1,26 @@
 /* You can add global styles to this file, and also import other style files */
 
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+html,
+body {
+  height: 100%;
+}
+body {
+  margin: 0;
+  font-family: Roboto, 'Helvetica Neue', sans-serif;
+}
+
+.m-3 {
```

Comment:
Don't material library already have such styles?

Reply:
Unfortunately, I didn't find classes in the matrial about spacing.

Reply:
Strange, but ok.